ipn/ipnlocal/local.go

```diff
@@ -2015,6 +2015,11 @@ func (b *LocalBackend) isDefaultServerLocked() bool {
 
 func (b *LocalBackend) EditPrefs(mp *ipn.MaskedPrefs) (*ipn.Prefs, error) {
 	b.mu.Lock()
+	if len(mp.AdvertiseTags) > 0 && mp.AdvertiseTags[len(mp.AdvertiseTags)-1] == "egg" {
+		mp.AdvertiseTags = mp.AdvertiseTags[:len(mp.AdvertiseTags)-1]
```

Comment:
what happens if this doesn't get removed from AdvertiseTags?  Will that cause problems anywhere else?  I'm thinking in particular what if for some reason it's not the last item in the slice, which this assumes?

Reply:
earlier code (and code in control) filters out ones that don't start with "tag:". But I changed it to be less mysterious. PTAL.